### ISSUE TITLE
VM boot failure on ubuntu

### ISSUE THREAD

**Issue by zhlhahaha** [kubevirt/kubevirt#5124]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
When booting example/vmi-fedora.yaml on **ubuntu**, it failed with following error message form virt-handler
```
{"component":"virt-handler","level":"error","msg":"failed to create tap device named tap0","pos":"podinterface.go:488","reason":"could not retrieve pid 24447 selinux label: operation not supported","timestamp":"2021-03-01T09:51:09.923220Z"}
{"component":"virt-handler","level":"error","msg":"failed to prepare pod networking","pos":"podinterface.go:204","reason":"could not retrieve pid 24447 selinux label: operation not supported","timestamp":"2021-03-01T09:51:09.923333Z"}
{"component":"virt-handler","kind":"","level":"error","msg":"Synchronizing the VirtualMachineInstance failed.","name":"vmi-fedora","namespace":"default","pos":"vm.go:1435","reason":"failed to configure vmi network: Critical network error: could not retrieve pid 24447 selinux label: operation not supported","timestamp":"2021-03-01T09:51:09.923729Z","uid":"066dccd6-7db2-428e-b53e-6484192155a4"}
{"component":"virt-handler","level":"error","msg":"virt-launcher crashed due to a network error. Updating VMI vmi-fedora status to Failed","pos":"vm.go:933","timestamp":"2021-03-01T09:51:09.924296Z"}
{"component":"virt-handler","level":"info","msg":"re-enqueuing VirtualMachineInstance default/vmi-fedora","pos":"vm.go:1043","reason":"failed to configure vmi network: Critical network error: could not retrieve pid 24447 selinux label: operation not supported","timestamp":"2021-03-01T09:51:09.956138Z"}
```

**What you expected to happen**:
VM boot successfully.

**How to reproduce it (as minimally and precisely as possible)**:
```
export KUBEVIRT_PROVIDER=kind-k8s-1.17
export KUBEVIRT_NUM_NODES=2
make cluster-up
make cluster-build
make cluster-deploy
kubectl apply -f example/vmi-fedora.yaml
kubectl logs -n kubevirt virt-handler-rhcts
```

**Anything else we need to know?**:
It seems related to the https://github.com/kubevirt/kubevirt/pull/4849/files.
Maybe because no selinux situation is not considered.

**Environment**:
- KubeVirt version: 7c366278fe76a6ec5cd8acacad7a7f3debef850e
- OS (e.g. from /etc/os-release): **Ubuntu 18.04.5 LTS**
- Kernel (e.g. `uname -a`): 5.9.0

**Comment by zhlhahaha** [comment]
cc @maiqueb @rmohr

**Comment by rmohr** [comment]
seems like https://github.com/kubevirt/kubevirt/pull/4849/files introduce this.

**Comment by vasiliy-ul** [comment]
I create a quick fix... for the case if this issue is blocking someone.

**Comment by wuestkamp** [comment]
ran into the same issue on ubuntu using `export RELEASE=v0.39.0`, downgrading to `export RELEASE=v0.38.1` solved it.

**Comment by maya-r** [comment]
There is a 0.39.1 release containing this fix, too.

**Comment by caohuilong** [comment]
I meet this error too, kubevirt(v0.54.0) with cilium CNI. @rmohr

**Comment by xpivarc** [comment]
> I meet this error too, kubevirt(v0.54.0) with cilium CNI in Debian11. @rmohr

Hi @caohuilong, Can you copy-paste your error?

**Comment by Viste** [comment]
same on debian 12 
`{"component":"virt-handler","level":"info","msg":"Generic Allocate: resourceName: tun","pos":"generic_device.go:242","timestamp":"2023-10-16T11:53:30.119235Z"}
{"component":"virt-handler","level":"info","msg":"Generic Allocate: request: [&ContainerAllocateRequest{DevicesIDs:[tun391],}]","pos":"generic_device.go:243","timestamp":"2023-10-16T11:53:30.119333Z"}
{"component":"virt-handler","kind":"","level":"info","msg":"VMI is in phase: Scheduled | Domain does not exist","name":"core-build-win01","namespace":"default","pos":"vm.go:1759","timestamp":"2023-10-16T11:53:32.560467Z","uid":"09415d56-18bd-4c4b-ab1e-da693a96630f"}
{"component":"virt-handler","kind":"","level":"info","msg":"mounting kernel artifacts","name":"core-build-win01","namespace":"default","pos":"mount.go:405","timestamp":"2023-10-16T11:53:32.563206Z","uid":"09415d56-18bd-4c4b-ab1e-da693a96630f"}
{"component":"virt-handler","kind":"","level":"info","msg":"kernel boot not defined - nothing to mount","name":"core-build-win01","namespace":"default","pos":"mount.go:408","timestamp":"2023-10-16T11:53:32.563263Z","uid":"09415d56-18bd-4c4b-ab1e-da693a96630f"}
{"component":"virt-handler","level":"error","msg":"failed to create tap device named tap0","pos":"bridge.go:142","reason":"could not retrieve pid 1514629 selinux label: operation not supported","timestamp":"2023-10-16T11:53:32.686078Z"}
{"component":"virt-handler","level":"error","msg":"failed to configure pod network: default","pos":"configstate.go:108","reason":"could not retrieve pid 1514629 selinux label: operation not supported","timestamp":"2023-10-16T11:53:32.686138Z"}
{"component":"virt-handler","kind":"","level":"error","msg":"Synchronizing the VirtualMachineInstance failed.","name":"core-build-win01","namespace":"default","pos":"vm.go:1905","reason":"failed to configure vmi network: setup failed, err: failed setup pod network phase1: Critical network error: could not retrieve pid 1514629 selinux label: operation not supported","timestamp":"2023-10-16T11:53:32.686215Z","uid":"09415d56-18bd-4c4b-ab1e-da693a96630f"}
{"component":"virt-handler","level":"error","msg":"couldn't find the SELinux context for core-build-win01","pos":"vm.go:1287","reason":"no data available","timestamp":"2023-10-16T11:53:32.707734Z"}
{"component":"virt-handler","level":"error","msg":"virt-launcher crashed due to a network error. Updating VMI core-build-win01 status to Failed","pos":"vm.go:1317","timestamp":"2023-10-16T11:53:32.707809Z"}
{"component":"virt-handler","level":"info","msg":"re-enqueuing VirtualMachineInstance default/core-build-win01","pos":"vm.go:1549","reason":"failed to configure vmi network: setup failed, err: failed setup pod network phase1: Critical network error: could not retrieve pid 1514629 selinux label: operation not supported","timestamp":"2023-10-16T11:53:32.749288Z"}`

**Comment by maiqueb** [comment]
@Viste if you're seening this again, please open a new issue and provide the data requested in the issue template.

Extremely important to know what node OS you're using, and which KubeVirt version you've deployed.
